--- /tmp/entropy/magneto/src/magneto/core/interfaces.py	2012-04-09 22:21:22.097174967 -0500
+++ /usr/lib/entropy/magneto/magneto/core/interfaces.py	2012-04-09 22:07:19.268151459 -0500
@@ -201,13 +201,13 @@
         )
 
     def show_service_available(self):
-        self.show_alert(
-            _("Sabayon updates service loaded"),
-            "%s: %s." % (
-                _("Entropy DBus service loaded"),
-                _("your Sabayon will notify you once updates are available"),
-            )
-        )
+       self.show_alert#(
+   #         _("Sabayon updates service loaded"),
+   #        "%s: %s." % (
+     #           _("Entropy DBus service loaded"),
+      #          _("your Sabayon will notify you once updates are available"),
+      #      )
+       # )
 
     def new_updates_signal(self):
         if not config.settings['APPLET_ENABLED']:
@@ -373,7 +373,7 @@
         subprocess.call(['xdg-open', url])
 
     def launch_package_manager(self, *data):
-        subprocess.call('sulfur &', shell = True)
+        subprocess.call('gksudo rigo &', shell = True)
 
     def disable_applet(self):
         self.update_tooltip(_("Updates Notification Applet Disabled"))
